.github/workflows/documentation.yml

```diff
@@ -7,6 +7,11 @@ on:
       - dependabot/**
   pull_request:
   workflow_dispatch:
+    inputs:
+      is-quarkus-nightly:
+        description: 'Testing Keycloak with the nightly Quarkus release'
+        required: false
+        default: false
```

Comment:
Maybe we don't need to add inputs to every workflow but only to `ci.yml` and `operator-ci.yml`. But I only assume the undefined input will resolve to `false` in `build-keycloak`.

Reply:
On the other hand, if we keep it in every workflow, we can then conditionally skip those. We don't need to test e.g. JS CI when `is-quarkus-nightly = true`.

Reply:
Alternatively, we can define a separate job in schedule-nightly.yml and then we can define a separate workflow matrix for it.

As of now, it is defined as a step, so it shares the strategy with the existing "Run workflow" step.

Reply:
Which actually got me to another thing - if we want to run it in parallel (I think we mentioned it during one of our discussions), then it is worth to define it as a job (instead of a step), because jobs run in parallel by default.

Afaik, it's possible to make even steps to run in parallel, but it requires an additional implementation.

Reply:
> if we want to run it in parallel (I think we mentioned it during one of our discussions), then it is worth to define it as a job (instead of a step), because jobs run in parallel by default

You mean the [step](https://github.com/keycloak/keycloak/blob/76e0e7e3f7886be819854dfaa2258c90fbadd688/.github/workflows/schedule-nightly.yml#L45-L48) in `schedule-nightly`? That's just a dispatch, it does not wait for the CI to complete before it dispatches another CI in the other step.